src/Table.h

```diff
@@ -258,7 +258,9 @@ class TableRow
         } else
           return ( noL && ! noR ? -1 : ! noL && noR ?  1 : 0);
       }
-      return ( a_r._columns[curr_column_r] < b_r._columns[curr_column_r] ? -1 : a_r._columns[curr_column_r] > b_r._columns[curr_column_r] ?  1 : 0 );
+      std::string lower_a_r = str::toLower(a_r._columns[curr_column_r]);
+      std::string lower_b_r = str::toLower(b_r._columns[curr_column_r]);
```

Comment:
Using `str::compareCI` (strcasecmp) would be the preferred way in such a  case because it avoids creating 2 string copies. But I'll close the PR because it will be solved by using new compare function, that will also ignore embedded ANSI SGR chars.